libraries/docker_container.rb

```diff
@@ -557,9 +557,9 @@ def load_container_labels
           } if new_resource.network_mode
           config.merge! net_config
 
-          config.merge(
```

Comment:
good catch - It was suggested by the linter.  it will actually add the healthcheck settings now.  I've updated the description of the PR.  There are zero tests for this configuration section of the code.  I am working in a branch to add tests so all the conditional statements in `create` will be fully tested.  hope to have that submitted very soon.

Reply:
@tas50 - hoping this can get merged so the subsequent PRs can be smaller if that's possible.
Thoughts?  Would it be better to handle in a different way?
The next PR will be to add tests around this container config - there are a lot of conditionals and it's too easy to make a mistake.